src/Abp/WebHooks/WebhookBody.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Abp.WebHooks
+{
+    public class WebhookBody
+    {
+        public string Event { get; set; }
```

Comment:
I think we can include `WebhookEventId` here

Reply:
`Event` property contains webhook's definition name which is unique webhook name.  
What do you mean by `WebhookEventId`?